### PR TITLE
Add Firewalls to Godotto

### PR DESCRIPTION
**Pull request by iheanyi**
This PR will be responsible for adding support for Firewalls into `godotto`.

### CODE REVIEW

File: pkg/loadbalancers/loadbalancers_test.go [hunk 1]
```diff
@@ -159,6 +159,7 @@ func TestLoadBalancerThrows(t *testing.T) {
 			{name: "get", fn: function() { pkg.get(lb.id) }},
 			{name: "delete", fn: function() { pkg.delete(lb) }},
 			{name: "list", fn: function() { pkg.list() }},
+            {name: "update", fn: function() { pkg.update(lb.id, lb) }},
```
Comment (aybabtme) [inline pkg/loadbalancers/loadbalancers_test.go]: indentation is off, (maybe space vs tabs?)

File: glide.yaml [hunk 2]
```diff
@@ -15,3 +15,5 @@ import:
   - context
 - package: golang.org/x/oauth2
 - package: gopkg.in/readline.v1
+- package: github.com/stretchr/testify
```
Comment (aybabtme) [inline glide.yaml]: idt this is used anywhere?

File: glide.lock [hunk 3]
```diff
@@ -63,4 +67,12 @@ imports:
   version: 6e83acea0053641eff084973fee085f0c193c61a
   subpackages:
   - base64vlq
-testImports: []
+testImports:
```
Comment (aybabtme) [inline glide.lock]: same, idt those are used anywhere

File: pkg/loadbalancers/loadbalancers_test.go (hunk 1 above)
Comment (iheanyi) [inline pkg/loadbalancers/loadbalancers_test.go]: @aybabtme  Yeah, thanks. I think emacs messed me up.

File: glide.lock (hunk 3 above)
Comment (iheanyi) [inline glide.lock]: I think these are deps from `testify/assert`.

**Comment by iheanyi** [comment]
Hey @aybabtme, this is ready for another round of review. As for adding in `testify/assert`, it's because it's hard to do a comparison of complex structs (structs in structs), especially with pointers involved so I added it in. Let me know your thoughts.

File: glide.lock (hunk 3 above)
Comment (aybabtme) [inline glide.lock]: `testify/assert` isnt being used in your code tho

**Comment by aybabtme** [comment]
:shipit:

**Pull request merged**